homework-04/config/config.json

```diff
@@ -1,3 +1,24 @@
 {
```

Comment:
Please, change this file to use .env variables. You can do it by renaming into `config.js`

Reply:
done